providers/src/airflow/providers/celery/executors/default_celery.py

```diff
@@ -69,6 +69,8 @@ def _broker_supports_visibility_timeout(url):
     log.debug("Value for celery result_backend not found. Using sql_alchemy_conn with db+ prefix.")
     result_backend = f'db+{conf.get("database", "SQL_ALCHEMY_CONN")}'
 
+extra_celery_config: dict = conf.getjson("celery", "extra_celery_config", fallback={})
```

Comment:
Now just a final static check failing, too much typing here as the json can produce the other types as well
```suggestion
extra_celery_config = conf.getjson("celery", "extra_celery_config", fallback={})
```